oriedita/src/main/java/oriedita/editor/swing/BottomPanel.java

```diff
@@ -59,6 +59,7 @@
     private JButton foldedFigureMoveButton;
     private UndoRedo undoRedo;
     private JComboBox<FoldedFigure_Drawer> foldedFigureBox;
+    private JButton constraintButton;
```

Comment:
Can you add constraintButton to the button index (with `buttonService.registerButton`, see line 87 of this file). And add an entry to help.properties with a short explanation.